README

```diff
@@ -641,11 +641,11 @@ OpenSHMEM Collectives
 Network Support
 ---------------
 
-- There are four main MPI network models available: "ob1", "cm",
-  "yalla", and "ucx".  "ob1" uses BTL ("Byte Transfer Layer")
+- There are several main MPI network models available: "ob1", "cm",
+  "ucx", and "yalla".  "ob1" uses BTL ("Byte Transfer Layer")
```

Comment:
pls remove mentions of mxm/yalla

Reply:
Wait; you just told me in an email that Mellanox wants to keep the MXM support in v4.0.0.

So I'm confused: do you want MXM support in 4.0.0 or not?

Reply:
can we keep it in the code base for now, but not document it in README, or document it as deprecated?

Reply:
Ah, I see.

Yes, I think we should mark it as deprecated.  If we leave it there undocumented, someone *will* go poking around and use it "just because it's there".  If we at least have it documented as deprecated, that will actually deter most people from using it.  I'll update the README and the web site PR (https://github.com/open-mpi/ompi-www/pull/117).

Reply:
@yosefe Is ikrit deprecated, too?

Reply:
yes